Sources/Purchasing/OfferingsManager.swift

```diff
@@ -213,10 +219,11 @@ extension OfferingsManager: @unchecked Sendable {}
 
 extension OfferingsManager {
 
-    enum Error: Swift.Error, Equatable {
```

Comment:
This didn't need to be `Equatable`. Small but useful code size decrease in the SDK 💪🏻